book/website/collaboration/research-infrastructure-roles/data-wrangler.md

```diff
@@ -0,0 +1,85 @@
+(cl-infrastructure-data-wranglers)=
+# Data Wranglers: Overview
+
+Data Wranglers can be viewed as a specialised type of Data Scientist, primarily working in the space between data generators and data analysts.
+The terminology used to describe various data science activities (for example data collection, wrangling, analysis, modelling, visualisation), and how they map onto different job titles, is domain specific and will vary on a project and organisational level.
+
+(cl-infrastructure-data-wranglers-tasks)=
+## What do Data Wranglers do?
+
+In a data science project, it is commonly observed that data wrangling tasks take the majority of time {cite:ps}`anaconda2020sods`, in contrast to data analysis and modelling.
+Traditionally, data wrangling tasks involve cleaning, restructuring and filtering data into standardised research ready formats.
```

Comment:
```suggestion
Traditionally, data wrangling tasks involve cleaning, restructuring and filtering data into standardised formats.
```
That or delete the standardised part and say ready for analysis?

Reply:
I like the focus on ready for analysis - data wranglers can also work in industry, not only in research

Reply:
Good point. We have definitely written this page with a bit of a research focus, but good to make the language more generalisable if possible

Resolved in [this commit](https://github.com/alan-turing-institute/the-turing-way/pull/2883/commits/11ad40708eb926bcaa325de61dccbc236d098160)